packages/redux/src/orders/selectors.ts

```diff
@@ -52,8 +63,15 @@ export const areOrdersLoading = (state: StoreState) =>
  *
  * @returns Orders error.
  */
-export const getOrdersError = (state: StoreState) =>
-  getError(state.orders as OrdersState);
+export const getUserOrdersError = (
+  state: StoreState,
+  query?: GetUserOrdersQuery,
+) => {
+  const hashedQuery = hashFetchUserOrdersQuery(query);
```

Comment:
As stated in another comment, you will need to add the `userId` to the request hash function as without it, the hash is not guaranteed to be unique among requests for different users.

Reply:
Changed it 🤝